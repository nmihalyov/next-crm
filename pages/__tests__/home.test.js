import HomePage from '../index';

describe('HomePage:', () => {
  it('renders home page', () => {
    const component = shallow(<HomePage />);

    expect(component).toMatchSnapshot();
  });
});