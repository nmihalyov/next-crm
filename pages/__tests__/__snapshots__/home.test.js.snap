// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage: renders home page 1`] = `
<MainLayout
  title="Main"
>
  <Row
    align="middle"
    justify="center"
  >
    <Title
      style={
        Object {
          "fontWeight": 900,
        }
      }
    >
      Welcome to Next CRM
    </Title>
  </Row>
  <Row
    align="middle"
    justify="center"
  >
    <Paragraph>
      A simple application based on React, NextJS, Redux & TypeScript
    </Paragraph>
  </Row>
</MainLayout>
`;
